eng/Tools.props

```diff
@@ -1,45 +1,15 @@
 <Project>
-  <PropertyGroup>
-    <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
-    <EnableAnalyzers>true</EnableAnalyzers>
-  </PropertyGroup>
-
-  <!-- We need to import this props file which contains PackageReferences to analyzers so that
-       analyzer build assets are imported via the Tools generated props and targets -->
-  <Import Project="$(RepositoryEngineeringDir)Analyzers.props" Condition="Exists('$(RepositoryEngineeringDir)Analyzers.props') and '$(DotNetBuildFromSource)' != 'true'" />
-
-  <!-- source-built packages -->
   <ItemGroup>
-    <!-- arcade -->
-    <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" />
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging" Version="$(MicrosoftDotNetBuildTasksPackagingVersion)" />
 
-    <!-- coreclr -->
     <!-- Download the package in the initial arcade restore step to work around race conditions when restoring an msbuild SDK. -->
     <PackageDownload Include="Microsoft.NET.Sdk.IL" Version="[$(MicrosoftNETSdkILVersion)]" />
-
-    <!-- roslyn -->
-    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="$(MicrosoftNetCompilersToolsetVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
-
-    <!-- mono -->
-    <PackageReference Include="illink.tasks" Version="$(ILLinkTasksVersion)" PrivateAssets="all" IsImplicitlyDefined="true" ExcludeAssets="build" GeneratePathProperty="true" />
-  </ItemGroup>
-
-  <!-- excluded from offline portion of source build -->
-  <ItemGroup Condition="'$(DotNetBuildOffline)' != 'true'">
-    <!-- arcade -->
-    <PackageReference Include="Microsoft.DotNet.GenAPI" Version="$(MicrosoftDotNetGenApiVersion)" />
   </ItemGroup>
 
   <!-- excluded from source build -->
   <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
-    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="$(MicrosoftDotNetApiCompatVersion)" />
```

Comment:
Where will we be getting all of these references now? I haven't found where all of these tools reference went, same for the compilers toolset and Genfacades.

Reply:
Oh nvm, I just found them 😄

Reply:
https://github.com/dotnet/runtime/pull/33242/files#diff-d3bedba39ce6da3e37ca0db83d7ef035R159-R164, and for the compiler https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Compiler.props#L5.